src/obb/server/http_component.clj

```diff
@@ -0,0 +1,39 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.http-component
+  "The aleph server as a component"
+  (:require [com.stuartsierra.component :as component]
+            [ring.middleware.cors :as cors]
+            [aleph.http :as http]
+            [compojure.core :as compojure]
+            [obb.server.routes :as routes]))
+
+(defn- setup-cors
+  "Setup cors"
+  [handler]
+  (cors/wrap-cors handler
+                  :access-control-allow-origin
+                  [#"^http://localhost(.*)"]
+                  :access-control-allow-methods [:get :put :post :delete]))
+
+(def app
+  "The main app handler"
+  (-> (compojure/routes routes/public-routes)
+      (setup-cors)))
+
+(defrecord HttpServerComponent [server port]
+
+  component/Lifecycle
+
+  (start [component]
+    (println (str "** OBB HTTP Server " (or (get (System/getenv) "OBB_ENV" "development"))
```

Comment:
This is a matter of taste, but I prefer not having defaults for configuration. I prefer that the application fails when a configuration is not set than having it work when I forgot to set an important variable. For example, if we forgot to set this on a staging deployment, it would still work but the environment would be set to development. This becomes more relevant for other configurations like database URIs or similar, but I tend to apply the same rule.

Having this said, it would be important to use something to help setting environment variables for the different environments like [environ](https://github.com/weavejester/environ). I know you normally use it. Did you just leave it for later?